src/System.IO.FileSystem/tests/Directory/GetLogicalDrives.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Linq;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Directory_GetLogicalDrives
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.AnyUnix)]
+        public void ThrowsPlatformNotSupported_Unix()
+        {
+            Assert.Throws<PlatformNotSupportedException>(() => Directory.GetLogicalDrives());
```

Comment:
Thanks. I didn't think mount points were similar enough to live in this API so I assumed (obviously incorrectly) that we didn't have an implementation.